### ISSUE TITLE
PreTrained on ImageNet

### ISSUE THREAD

**Issue by Lwt-diamond** [thograce/C2FNet#8]
Did you used Res2Net pretrained on ImageNet as a backbone ? But the input size of the Res2Net which pretrained on ImageNet  is 224x224. The input size in your code( which is also named train_size ) is 352x352. In my opinion, the pretrained data  on ImageNet is not suitable for the model.  Could you tell me your opinion? Thank you very much

![image](https://user-images.githubusercontent.com/82935744/222916694-5e6bcaf0-68a5-4f73-9734-174b27c93a3d.png)

**Comment by thograce** [comment]
> Did you used Res2Net pretrained on ImageNet as a backbone ? But the input size of the Res2Net which pretrained on ImageNet is 224x224. The input size in your code( which is also named train_size ) is 352x352. In my opinion, the pretrained data on ImageNet is not suitable for the model. Could you tell me your opinion? Thank you very much
> 

Yes, I used Res2Net pre-trained on ImageNet as a backbone. Now let me explain the selection of training image input size. 
1.The general baseline method for camouflage object detection (COD), SINet, selects 352x352 as the input size. For fair comparison, many subsequent methods follow the input size of SINet, so does C2FNet. 
2.COD is not a classification task. Res2Net is only used as a feature extractor, not a classifier, so the FC layer is abandoned. Therefore, the change of input size will not affect the network operation.
3.COD is a pixel-level segmentation task, which requires high resolution to retain more effective information. The datasets represented by COD10K all have very high resolution, so it is obviously inappropriate to compress all images to 224x224. 
4.Relevant research also shows that improving resolution can achieve greater benefits. You can see the change of input size during the evolution of YOLO series, which will help you understand transfer learning and finetuning.
5. In general, for non-classification tasks, it is a domain consensus that the input size is not limited by the pre-trained model.
